### ISSUE TITLE
Multi-tenancy flags are inconsistently documented

### ISSUE THREAD

**Issue by akeller** [camunda/camunda-docs#2990]
We inconsistently refer to enabling multi-tenancy as a feature flag, environment variable, or unspecified config:

- [Identity feature flags](https://docs.camunda.io/docs/self-managed/identity/deployment/configuration-variables/#feature-flags)
- [Zeebe multi-tenancy](https://docs.camunda.io/docs/self-managed/zeebe-deployment/configuration/gateway-config/#zeebegatewaymultitenancy)
- [Operate multi-tenancy](https://docs.camunda.io/docs/self-managed/operate-deployment/operate-configuration/#multi-tenancy)
- [Tasklist multi-tenancy](https://docs.camunda.io/docs/self-managed/tasklist-deployment/tasklist-configuration/#multi-tenancy)
- [Optimize multi-tenancy](https://docs.camunda.io/optimize/self-managed/optimize-deployment/configuration/multi-tenancy/)
- [Connectors multi-tenancy](https://docs.camunda.io/docs/self-managed/connectors-deployment/connectors-configuration/#multi-tenancy)

**Recommended action** - change link text to just the component name. 

The following includes the current patterns and recommendations:
* Identity - Environment variable | Description | Default Value (false)
* Zeebe - Field | Description | Example Value (True), as a YAML example
  * Example value field matches the rest of the page, however it doesn't match the rest of the component docs
  * **Recommended action** - set the example value as the default value - `false` (lowercase, see below) 
* Operate - Name | Description | Default Value (false)
* Tasklist - YAML Path | Environment variable | Default Value | Description
  * Doesn't match the other tables on the page (Name | Description | Default value)
  * **Recommend action** - match the page and include the environment variable outside of the table since the page state "by default, the configuration for Tasklist is stored in a YAML file application.yml"
* Optimize (labeled C7 only)
  * Where are the Optimize + MT docs for C8? 
  * The first full sentence should be rewritten. Both C7 and C8 docs live in docs.camunda.io for now.
* Connectors - Name/Description/Default value (`<default>`)
  * Is this really the default value? 
  
The casing for default or example values is inconsistent - is it correct as documented? Or is it `true` or `false` (all lowercase)? 

* Identity, Operate, Tasklist - lowercase
* Zeebe - first letter uppercase in the table, but lowercase in the YAML snippet
* Optimize - Not documented?
* Connectors - `<default>`

[Internal Slack thread.](https://camunda.slack.com/archives/C0555930H7F/p1701816113847239)

_Raised by @pierre-yves-monnet via DM. Thank you!_

**Comment by PHWaechtler** [comment]
> Optimize (labeled C7 only)
> * Where are the Optimize + MT docs for C8?
> * The first full sentence should be rewritten. Both C7 and C8 docs live in docs.camunda.io for now.

@akeller It looks like because I created the Optimize PR before the release, but this was merged into the MT docs branch which was merged in to the docs late, there was no backporting of this Optimize section for 8.3 (which looks like it was still "next" when the PR was merged, but outdated once the entire MT docs were merged).
Can you please review the Optimize section as is when selecting "next"? I will create a PR to backport correctly.

**Comment by akeller** [comment]
@PHWaechtler thank you! I will also look at the other component docs to make sure they are properly backported as well.

**Comment by akeller** [comment]
Thread opened for Connectors - https://camunda.slack.com/archives/C0555930H7F/p1701875344024639

**Comment by akeller** [comment]
Moving Connectors to a separately scoped issue to stay unblocked.
